bunsen-avro/src/main/java/com/cerner/bunsen/avro/converters/DefinitionToAvroVisitor.java

```diff
@@ -405,7 +406,10 @@ public HapiFieldSetter toHapiConverter(BaseRuntimeElementDefinition... elementDe
 
   private String recordNameFor(String elementPath) {
 
-    return elementPath.replaceAll("\\.", "");
+    return Arrays.stream(elementPath.split("\\."))
+        .map(StringUtils::capitalize)
```

Comment:
there's a pre-existing custom implementation of capitalize down below: https://github.com/cerner/bunsen/pull/59/files#diff-59b60ed682a9e8e75a39b15530d028d9R605

Probably should only keep 1 implementation in use.  I suggest removing the custom implementation and instead use the StringUtils implementation in the 1 place below where that is currently being used.

Reply:
Removed the custom implementation in favor of the `StringUtils` library, 44b18b5838cf49bff094330b7ebbe4092c67362f